app/scripts/services/quota.js

```diff
@@ -277,6 +280,50 @@ angular.module("openshiftConsole")
         return _.some(quotas, isExceeded) || _.some(clusterQuotas, isExceeded);
     };
 
+    //Specific to storage quotas for max claim count and max storage usage
+    // and the amount of claims and storage already used
+
+    var getStorageQuotas = function(quotas,clusterQuotas) {
+      // Currently, there can be multiple quotas per namespace
+      // So let's go through them and find the most restrictive
+      var storageQuotas = {};
+
+      var getMostRestrictive = function(quotas) {
+        if (_.isEmpty(quotas)) {
+           return;
+        }
+        _.each(quotas, function(quota) {
+          //Need to find the minimum quotas
+          var q = quota.status.total || quota.status;
+          if (q.hard.persistentvolumeclaims) {
+            if ((!storageQuotas.maxNumberClaims ) || (parseInt(q.hard.persistentvolumeclaims) < storageQuotas.maxNumberClaims)) {
+              storageQuotas.maxNumberClaims = parseInt(q.hard.persistentvolumeclaims);
+            }
+          }
+          if (q.hard['requests.storage']) {
+            if ((!storageQuotas.maxStorage) || (usageValue(q.hard['requests.storage']) < usageValue(storageQuotas.maxStorage))) {
+              storageQuotas.maxStorage = q.hard['requests.storage'];
+            }
+          }
+
+          //Get the amount used so far
+          var used = usedStorage(quota);
```

Comment:
this logic isn't going to work if you've got multiple cluster quotas, imagine you've got projects A, B, and C.  clusterquota-1 covers projects A and B, clusterquota-2 covers projects B and C.   Consume some storage resources in project A, consume a different amount of storage resources in C.  If you are then in project B you will see two clusterquotas with different amounts of "used" quota.